.github/workflows/standard_spa_pr.yml

```diff
@@ -452,7 +452,9 @@ jobs:
         if: ${{ inputs.e2e_pass_on_error }}
         run: exit 0
       - name: Set success vars
-        if: ${{ contains(steps.cypress_run.outcome, 'success') }}
+        # Need to check previous TEST_STATUS_STATE to ensure it hasn't already failed
+        # important if multiple containers are in use to ensure the correct status is reported
+        if: ${{ contains(steps.cypress_run.outcome, 'success') && !contains(env.TEST_STATUS_STATE, 'failure') }}
```

Comment:
Addresses https://jupiterone.atlassian.net/browse/APP-13239, ensuring the proper status is reported in the PR when multiple containers are used.